### ISSUE TITLE
Add Introduction to Dynamic Stream Handling

### ISSUE THREAD

**Issue by muskan3006** [akka/akka#30720]
![image](https://user-images.githubusercontent.com/48651303/134824554-c76d93db-3a49-4e0d-9a71-7fd2e980e41a.png)

Here it can be seen that we don't have any introduction for the topic Dynamic Stream Handling. I think it can be more helpful if we have some details there.

Thanks!

**Comment by johanandren** [comment]
I agree, a short bit about how Akka streams blueprints are generally re-usable but static and perhaps describing what dynamic means in that context would be good.
